homeassistant/components/ozw/climate.py

```diff
@@ -174,7 +174,8 @@ def on_value_update(self):
     def hvac_mode(self):
         """Return hvac operation ie. heat, cool mode."""
         if not self.values.mode:
-            return None
+            # Thermostat(valve) with no support for setting a mode is condidered heating-only
+            return HVAC_MODE_HEAT
```

Comment:
Would this also fix #28834 and perhaps also an issue related to #34788 but for Z-Wave devices? As I still cannot modify the temperature of my climate devices with the old Z-Wave component.

Reply:
correct